### ISSUE TITLE
Cannot open the generated ios workspace

### ISSUE THREAD

**Issue by zabojad** [frostney/react-native-create-library#95]
We cannot open the generated ios workspace.

`react-native-create-library@3.1.2`

XCode 10.0

**Comment by zabojad** [comment]
Ok my bad, it works with master version... Then maybe a new release would be needed?

**Comment by worldofnick** [comment]
Yeah, looks like we need a new release. @zabojad can you share how you installed the master version? Thanks :)

**Comment by zabojad** [comment]
@worldofnick `npm install -g git+https://github.com/frostney/react-native-create-library.git` should do the trick.

**Comment by maicki** [comment]
FYI, we track the release in here: https://github.com/frostney/react-native-create-library/issues/78

**Comment by SaeedZhiany** [comment]
@zabojad 
Is there any difference between the template created by master version and the one created by normal way (I mean the way we install this library using npm)
I'm really confused!

**Comment by neurosnap** [comment]
Any updates on this?  Installing from github is fine but it's been months, can we get this published?
